### PR TITLE
[Model Enablement] fix default onnx domain

### PR DESCRIPTION
**Pull request by bszmelcz**
### Details:
 - [x] unify default domain to empty string
 - [x] fix 2 models containing Unsqueeze operation 
### Tickets:
 - 70298
 - 70300

### CODE REVIEW

**Review by mitruska — COMMENTED** [review]
Nice to see it can be easily fixed, please add unit tests to cover the issue case.

**Review by tomdol — CHANGES_REQUESTED** [review]
As already suggested, please add a unit test. A simple one with ai.onnx domain and a single Unsqueeze operator will do.

File: src/core/tests/models/onnx/unsqueeze_ai_onnx_domain_opset13.prototxt [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          dim {
+            dim_value: 5
+          }
+        }
+      }
+    }
+  }
+}
+opset_import {
+  domain: ""
```
Comment (mitruska) [inline src/core/tests/models/onnx/unsqueeze_ai_onnx_domain_opset13.prototxt:81]: It seems the intention of this test was to check the "ai.onnx" domain, 
but the empty string is tested like in the default_domain test:
```suggestion
  domain: "ai.onnx"
 ```

**Review by mitruska — CHANGES_REQUESTED** [review]

File: src/core/tests/models/onnx/unsqueeze_ai_onnx_domain_opset13.prototxt (hunk 1 above)
Comment (bszmelcz) [reply src/core/tests/models/onnx/unsqueeze_ai_onnx_domain_opset13.prototxt]: Oops, forgot to change it, thanks :)

**Review by mitruska — APPROVED** [review]

**Review by tomdol — APPROVED** [review]

**Review by mbencer — APPROVED** [review]

**Pull request merged**